jsonschema.go

```diff
@@ -0,0 +1,311 @@
+package generate
```

Comment:
It's a shame this isn't picked up as a move. A few people have worked on it, as a delete and create, then it might not pick up their names as authors. I'm not sure.

Reply:
$ git log --follow jsonschema.go
... returns the full history, not sure why it doesn't get transferred from the rename... 
The blame log remains the same though. Could start an AUTHORS or ChangeLog file?